Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -2451,3 +2451,71 @@ M  END
     outs.flush();
   }
 }
+
+TEST_CASE("test the options that toggle isotope labels", "[drawing]") {
+  SECTION("test all permutations") {
```

Comment:
same question here about simplifying these tests and making them more direct

Reply:
Done - I am now counting matches to a simple regex.
The JS still uses the sort as `noFreetype` is not exposed and I still want a test for this functionality.